glossary.md

```diff
@@ -19,4 +19,13 @@
 **Service provider**: The entity using Lock Keeper to build a full-featured digital asset manager.
 
 
+## Technical Definitions
+
+**Workflow/Flow**: A set of potentially non-linear steps that accomplish an action on behalf of a user. Completing a workflow requires the execution of one or more protocols. *Example: Generate and store a secret*
```

Comment:
We could use "endpoint" or "method" instead of "workflow". I think "method" is the most correct term for gRPC but it's pretty common for people to call them "endpoints" out of habit. "Endpoint" is the REST term but we're not using REST here.

I don't think there's any non-linearity right now but I was thinking there might be in the future. I think we can remove that word if we're going to keep this definition.

Reply:
Given that we already use "endpoint", we probably want to either stick to workflow for now.